src/css/components/grid.scss

```diff
@@ -1,7 +1,12 @@
 .grid {
-  @include outer-container;
```

Comment:
I actually had to change this separately in my footer pr, https://github.com/18F/cg-style/pull/275. We can figure it out once they get merged together.